cmd/hub/cli/pow.go

```diff
@@ -0,0 +1,249 @@
+package cli
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/spf13/cobra"
+	ffsRpc "github.com/textileio/powergate/ffs/rpc"
+	"github.com/textileio/textile/cmd"
+)
```

Comment:
Simple CLI for Powergate-related calls, all commands live under `hub pow`. Keeping it as simple as possible for now, just calling the methods and printing out the response.